ppl/zqd/pcapanalyzer/process.go

```diff
@@ -28,27 +31,40 @@ func wrapError(err error, name, stderr string) error {
 	}
 	var pathErr *os.PathError
 	if errors.As(err, &pathErr) {
-		return fmt.Errorf("error executing %s runner: %s: %v", name, pathErr.Path, pathErr.Err)
+		return fmt.Errorf("error executing %s: %s: %v", name, pathErr.Path, pathErr.Err)
 	}
 	return err
 }
 
 type Process struct {
 	cmd       *exec.Cmd
 	stderrBuf *bytes.Buffer
+	stdoutBuf *bytes.Buffer
 }
 
 func NewProcess(cmd *exec.Cmd) *Process {
-	p := &Process{cmd: cmd, stderrBuf: bytes.NewBuffer(nil)}
-	// Capture stderr for error reporting.
+	p := &Process{cmd: cmd, stderrBuf: bytes.NewBuffer(nil), stdoutBuf: bytes.NewBuffer(nil)}
 	cmd.Stderr = p.stderrBuf
 	return p
 }
 
+func NewProcessStdout(cmd *exec.Cmd) *Process {
```

Comment:
How about a `stdout bool` parameter to `NewParameter`?

Reply:
Sure.

Reply:
(I'm assuming you meant NewProcess).